tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -209,6 +209,17 @@ Verify CPU And Memory Requests And Limits Are Defined For All Containers In All
     Verify CPU And Memory Requests And Limits Are Defined For All Containers In All Pods In Project    redhat-ods-monitoring
     Verify CPU And Memory Requests And Limits Are Defined For All Containers In All Pods In Project    redhat-ods-operator
 
+Verify Monitoring Stack Is Reconciled Without Restarting The ODS Operator
+    [Documentation]    Verify Monitoring Stack Is Reconciled Without Restarting The RHODS Operator
+    [Tags]    Sanity
+    ...       Tier2
+    ...       ODS-699
+    OpenShiftCLI.Patch    kind=ServiceMonitor
+    ...                   src={"spec":{"selector":{"matchLabels": {"app":"grafana"}}}}
+    ...                   name=rhods-monitor-federation   namespace=redhat-ods-monitoring  type=merge
+    Sleep    10m    msg=wait to operator reverts the Changes
+    Wait Until Operator Reverts "Grafana" To "Prometheus" In Rhods-Monitor-Federation
```

Comment:
I think these two lines are redundant. Line 221 is already waiting, why using a sleep? You could increase the timeout in `Wait Until Operator Reverts "Grafana" To "Prometheus" In Rhods-Monitor-Federation`

Reply:
@bdattoma 
After approx 10 min it starts to revert the changes in the first 10 min operator does not do anything so if we check for changes that will be a waste of requests.
so that is why I put them to sleep for 10 min.
polarion id = ODS-699
in polarion, 5. check if the operator reverts the change after approx 10 minutes

Reply:
hm okay but if you wait for 10 minutes at this point, why then you set a timeout of 5minutes in the `Wait Until Keyword Succeeds`? 

However, if the sleep is needed, then I would move line 218 to `Wait Until Operator Reverts ...` since it is part of waiting process. 

Last thing. Have you manually tried the process to check the timing? I put 10 minutes approx but maybe we could optimize it for the automation so we waste less time

Reply:
Sure I will move sleep to ` wait until operator ..` 
and for the first question revert operation start after 10 min 
yesterday while testing I saw that it sometimes takes 1 min or  2 min (After completing 10 mins) so that is  I put a timeout of 5min.
Yes  I have tested it first manually that time I have checked the time also
steps / time
` Replace "Prometheus" With "Grafana" In Rhods-Monirot-Federation ` run at 16:24 (IST)
` Verify In Rhods-Monitor-Federation App Is expected_app_name=prometheus ` run at 16:34(IST) but it failed
` Verify In Rhods-Monitor-Federation App Is expected_app_name=prometheus ` run at 16:35(IST)         PASS

Reply:
gotcha. Sounds good then to me

Reply:
Thanks @bdattoma